assets/js/src/notices/api_errors_notice.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+import Notice from 'notices/notice';
+
+type Props = {
+  errors: Array<{ message: string }>
+}
+
+const APIErrorsNotice: FC<Props> = ({ errors }) => {
```

Comment:
Is it necessary to import and use `FC` in all functional components now? Or can we do something like this (https://github.com/milesj/babel-plugin-typescript-to-proptypes):
```
const APIErrorsNotice: (props: Props) => {
  const errors = props.errors;
```

Or is there a difference? Just trying to figure out what will be the best practices now.

Reply:
The two ways are equivalent, the only difference is that using `FC<Props>` will implicitly add an optional `children` property of type `ReactNode`.